.circleci/config.yml

```diff
@@ -35,6 +35,7 @@ jobs:
                     . venv/bin/activate
                     pip install -r requirements-base.txt --quiet
                     pip install -r requirements-v0.txt --quiet
+                    pip install -e .
```

Comment:
Might be better to put this before the `install -r` items? As it is seems like `install -r` will grab `dash-table@3.3.0` each time, then `install -e` will put back the local version... same result but should be faster if we tell it up front we're using local `dash-table`.

Reply:
Sure. If it doesn't get overriden, makes sense!